wandb/integration/keras/keras.py

```diff
@@ -786,16 +786,38 @@ def _log_weights(self):
         for layer in self.model.layers:
             weights = layer.get_weights()
             if len(weights) == 1:
-                metrics["parameters/" + layer.name + ".weights"] = wandb.Histogram(
-                    weights[0]
-                )
+                if is_numeric_array(weights[0]):
+                    metrics["parameters/" + layer.name + ".weights"] = wandb.Histogram(
+                        weights[0]
+                    )
+                else:
+                    wandb.termwarn(
```

Comment:
termwarn(repeat=False) probably.  or are layer names always unique?

Reply:
Keras creates an error if all layer names are not unique.